### PR TITLE
Rev dependencies. Fix npm audit grumps.

### PR DESCRIPTION
**Pull request by robdodson**
Fixes #4462, fixes #478

Revs npm deps to latest versions.

### CODE REVIEW

**Comment by robdodson** [comment]
ok I think this might be more complicated than I anticipated. I think revving the remark linters sort of "activated" them and now they're flagging a number of files that might need to be tweaked.

File: src/site/content/en/blog/building-a-tabs-component/index.md [hunk 1]
```diff
@@ -468,7 +468,7 @@ styles here, I just need to define how it overflows. I set overflow-y to auto,
 and then also trap the scroll interactions with the handy overscroll-behavior
 property.
 
-#### 3 scroll areas recap {: #overscroll }
+#### 3 scroll areas recap {: #scroll-areas-recap }
```
Comment (samthor) [inline src/site/content/en/blog/building-a-tabs-component/index.md:471]: This seems unrelated.

Comment (samthor) [reply src/site/content/en/blog/building-a-tabs-component/index.md:471]: Oh, I get it, the linter is now complaining.

Comment (samthor) [reply src/site/content/en/blog/building-a-tabs-component/index.md:471]: The `no-repeat-punctuation` thing seems like a bug—it's catching strikethrough.

**Comment by samthor** [comment]
Closing this since really it needs updating as deps are changing from underneath us.
